pepper-apis/studybuilder-cli/src/main/java/org/broadinstitute/ddp/studybuilder/task/rootpatches/Osteo2pecgsUpdates.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.ddp.studybuilder.task.rootpatches;
```

Comment:
One root patch to run in other env's (non-dev) as part of deploy

Reply:
Amazing idea!